fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryLoadByKeysPlan.java

```diff
@@ -242,7 +242,8 @@ public int hashCode() {
         }
 
         @Override
-        public int planHash() {
+        public int planHash(PlanHashKind hashKind) {
+            // TODO: Right?
```

Comment:
ok

Reply:
Hmm -- I think I am torn here. There is not an actual problem here, more just a convention that we should zone in on. If possible should we call `hashCode()` or should we separate this out and even go so far as to duplicate the code. This here only works because the plan operator by itself is a leaf in the graph. Otherwise